app/(editor)/editor/page.tsx

```diff
@@ -86,6 +95,65 @@ async function getStoryData({ path }: PageProps['searchParams']): Promise<ISbRes
 
 };
 
+/**
+ * Get a list of stories that are of component sbStoryMvp in reverse chronological order.
+ */
+async function getStoryList({ path }: PageProps['searchParams']) {
+  const activeEnv = process.env.NODE_ENV || 'development';
+  const storyblokApi: StoryblokClient = getStoryblokApi();
+  const fullslug = path.replace(/\/$/, '') || 'home';
+
+  // Get the last part of the path.
+  const slug = path.split('/').pop() || '';
+
+  let orQuery: FilterQuery[] = [];
+
+  /**
+   * If the page is inside the folder stories/list/ (story list pages filtered by taxonomy),
+   * add a filter query to only return stories that has an initiative or theme that matches the slug of that story.
+   * E.g., if the full slug is stories/list/preparing-citizens,
+   * only return stories that have 'preparing-citizens' tagged as a theme or initiative.
+   */
+  if (fullslug.includes('stories/list/')) {
+    orQuery = [
+      {
+        initiatives: {
+          in_array: slug,
+        },
+      },
+      {
+        themes: {
+          in_array: slug,
+        },
+      },
+    ];
+  }
+
+  // For more related documentation see app/(storyblok)/[[...slug]]/page.tsx
+  const sbParams: ISbStoriesParams = {
+    version: 'published',
+    cv: activeEnv === 'development' ? Date.now() : undefined,
+    starts_with: 'stories/',
+    sort_by: 'first_published_at:desc',
+    per_page: 100,
+    filter_query: {
+      component: {
+        in: 'sbStoryMvp',
+      },
+      __or: orQuery,
```

Comment:
`__or` is how we can add query using the or operator. So here either the slug matches an item in the initiatives, or in the themes.